common/scripts/bump-beta-release-version.js

```diff
@@ -31,6 +31,12 @@ const bumpBetaVersion = (currentVersion) => {
     }
     const newBeta = Number.parseInt(betaVersion) + 1;
     return `${major}.${minor}.${patch}-beta.${newBeta}`
+  } else if (bumpType === 'stable') {
+    // if bumpType is stable we want to reset the beta version
+    const newMinor = Number.parseInt(minor) + 1;
```

Comment:
Why don't we update L29 to update the minor version instead of the patch - when would we still want to go to `x.y.z+1-beta.0`?

Reply:
I tried this on the PR and the issue is this then fails when bumping the dep in the sample. it calls back to line 32 so I thought making a new mode of the function would fix this.